src/main.js

```diff
@@ -15,13 +16,12 @@ handleLocation();
 
 updateQuote();
 
-document.addEventListener('DOMContentLoaded', async () => {
-  if (window.location.pathname === '/') {
-    renderCards();
-  }
-});
+// document.addEventListener('DOMContentLoaded', async () => {
+//   if (window.location.pathname === '/') {
+//     renderCards();
+//   }
+// });
 
```

Comment:
renderCards call should go to the apiCallback, which will be called in route funcion, and on load with default params. Please keep code clean and remove commented code

Reply:
fix